docs/guide/apm-data-model.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[apm-data-model]]
+== Data Model
+
+Elastic APM agents capture different types of information from within their instrumented applications - namely `transactions`, `spans`, and `errors`. 
+
+* <<transactions,Transaction>>
+* <<transaction-spans,Spans>>
+* <<errors,Errors>>
+
+[[transactions]]
+=== Transactions
+
+A transaction describes an event captured by an Elastic APM agent instrumenting a service. Some examples of a transaction might be:
```

Comment:
I don't know if this is just moved over, but thinking about it now we are defining things in terms of each other:
"transactions, spans and errors are types of events"
" a transaction describes an event..."

i don't have better ideas, but maybe explain-me-like-i-am-5 style?
"a transaction has an start and and end, and some information describing what happens in between..."

Reply:
Good call - some of the copied over text doesn't quite make sense.  I'll take a better look at this once I get the other changes added